oauth-client-credentials.html.md.erb

```diff
@@ -0,0 +1,54 @@
+---
+title: OAuth Client Credentials 
+owner: API
+---
+
+<strong><%= modified_date %></strong>
+
+## <a id='overview'></a> OAuth Client Credentials Tutorial
+
+**Note: The Swisscom OAuth is only usable if it is approved and granted from the API team - contact us via the [contact form](https://developer.swisscom.com/support)**
+
+This tutorial describes an easy way to get an OAuth authorization code & access token and shows the integration flow.
+
+Tools:
+
+* Terminal: to make curl calls
+* Browser: https://www.base64encode.org/
+
+### 1. Encode your client id and your client id secret
+
+Go to the page https://www.base64encode.org/ and enter %YOUR_CLIENT_ID%COLON%YOUR_CLIENT_ID_SECRET% and click > ENCODE < .
+
+![Encode your credentials](img/encodecc.png)
+
+Use this result for the authorization in step 2.
+
+#### 2. Get the Access token
+
+Open you terminal and type
+
+```
+curl -ik X POST "https://consent.swisscom.com/o/oauth2/token" -H "Authorization: Basic %YOUR_CLIENT_ID%:%YOUR_CLIENT_ID_SECRET%(As BASE64)" -d "grant_type=client_credentials"
+```
+
+Use the result of step 1 to replace %YOUR_CLIENT_ID%:%YOUR_CLIENT_ID_SECRET%(As BASE64)
+
+Response Payload: 
+
+```JSON
+{
+    "access_token": "KASDFHDASHFSASHAS_DHAISDHFSAIDS",
+    "scope": "scope1 scope2",
+    "token_type": "bearer",
+    "expires_in": "2591999"_
+}
+```
+
+#### 3. Ready to use the Access token
+
+As example: Voice Voip Numbers as a curl call
+
+```
+curl -ik  -H "Authorization: Bearer KASDFHDASHFSASHAS_DHAISDHFSAIDS" -X GET "https://api.swisscom.com/v1//voice/v1/voip/me/numbers"
```

Comment:
is `https://api.swisscom.com/v1//voice/v1/voip/me/numbers` really the correct URL?